### ISSUE TITLE
breadcrumb

### ISSUE THREAD

**Issue by cedk** [openlabs/tryton.us#128]
The breadcrumb should be always visible, probably in the navbar or under.

**Comment by rituparnapanda** [comment]
@cedk Could you please explain a bit more about this issue ? Breadcrumb is already there. 

![screenshot 2015-02-26 20 15 52](https://cloud.githubusercontent.com/assets/2655038/6393985/6d03c6e4-bdf4-11e4-93d0-df738484323e.png)

**Comment by cedk** [comment]
Yes but the place is not good because it doesn't have a fixed position so when you scroll down, you can no more see it and so it is difficult to switch to the upper level for example.
I would put the breadcrumb at a fixed position on top with the navbar (or in the navbar)
Example: http://stackoverflow.com/questions/20270438/breadcrumb-into-a-navbar-in-bootstrap-3-0-2
